src/output.rs

```diff
@@ -83,13 +83,15 @@ impl OutputType {
             let replace_arguments_to_less = pager.source == PagerSource::EnvVarPager;
 
             if args.is_empty() || replace_arguments_to_less {
-                p.arg("--RAW-CONTROL-CHARS");
+                if args.is_empty() || replace_arguments_to_less {
+                    p.arg("-R");
```

Comment:
Let's save others the trouble of looking up what `-R` means by changing to this:

```suggestion
                    p.arg("-R"); // Short version of --RAW-CONTROL-CHARS for maximum compatibility
```

Same for the other short options.